lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -16,6 +16,8 @@
  */
 'use strict';
 
+/* eslint-disable no-console */
```

Comment:
since there's only one `console.log` in here for a specific reason, but it's mostly a `log.*()` file, does it make sense to limit to `// eslint-disable-next-line no-console` over line 285?